src/test/java/org/apache/commons/jxpath/ri/JXPathContextReferenceImplTestCase.java

```diff
@@ -29,12 +31,10 @@ public class JXPathContextReferenceImplTestCase {
     @Test
     public void testInit() {
         final ContainerPointerFactory factory = new ContainerPointerFactory();
-        try {
-            JXPathContextReferenceImpl.addNodePointerFactory(factory);
-        } finally {
-            while (JXPathContextReferenceImpl.removeNodePointerFactory(factory)) {
+        assertDoesNotThrow(() -> JXPathContextReferenceImpl.addNodePointerFactory(factory));
```

Comment:
Same comment as before RE `assertDoesNotThrow()`.

Reply:
Since `addNodePointerFactory` does not return anything, what is to be asserted here? Or should I simply remove the assertion and leave the statement as is?

Reply:
Then don't call assertDoesNotThrow() and that's it 👍